README.md

```diff
@@ -1,55 +1,361 @@
 
-The following packages are included in Fury Kubernetes Disaster Recovery modules:
+If you are new to KFD please refer to the [official documentation][kfd-docs] on how to get started with KFD.
 
-- [aws-velero](modules/aws-velero): Creates AWS resources and Kubernetes CRDs needed to persist backups.
-- [eks-velero](modules/eks-velero): Creates AWS resources and Kubernetes CRDs needed to persist backups
-from an eks cluster.
-- [azure-velero](modules/azure-velero): Creates Azure resources and Kubernetes CRDs needed to persist backups.
-- [gcp-velero](modules/gcp-velero): Creates GCP resources and Kubernetes CRDs needed to persist backups.
+## Overview
 
+**Kubernetes Fury DR** module is based on [Velero][velero-page] and Velero Restic.
+
+Velero allows you to:
+
+- backup your cluster
+- restore your cluster in case of problems
+- migrate cluster resources to other clusters
+- replicate your production environment to development and testing environment.
+
+Together with Velero, Velero Restic allows you to:
+
+- backup Kubernetes volumes
+- restore Kubernetes volumes
+
+The module contains also velero plugins to natively integrate with Velero with different cloud providers and use cloud provider's volumes as the storage backend.
+
+## Packages
+
+Kubernetes Fury DR provides the following packages:
+
+|                      Package                      | Version |                                   Description                                    |
+| ------------------------------------------------- | ------- | -------------------------------------------------------------------------------- |
+| [velero](katalog/velero)                          | `1.7.1` | Backup and restore, perform disaster recovery, and migrate Kubernetes cluster resources and persistent volumes. |
+
+The velero package contains the following additional components:
+
+|                     Component                     |                      Description                      |
+| ------------------------------------------------- | ----------------------------------------------------- |
+| [velero-restic](katalog/velero/velero-restic)     | Incremental backup and restore of Kubernetes volumes. |
+| [velero-schedule](katalog/velero/velero-schedule) | Common schedules for backup                           |
+
+### Integration with cloud providers
+
+Use the following Velero Plugins to integrate Velero with cloud providers:
+
+|                   Plugin                    |                      Description                      |
+| ------------------------------------------- | ----------------------------------------------------- |
+| [velero-aws](katalog/velero/velero-aws)     | Plugins to support running Velero on AWS  |
+| [velero-gcp](katalog/velero/velero-gcp)     | Plugins to support running Velero on GCP |
+| [velero-azure](katalog/velero/velero-azure) | Plugins to support running Velero on Azure |
```

Comment:
missing nice formatting in the las column :-P

Reply:
OCD-ed!